core/templates/dev/head/pages/library/library_footer_directive.html

```diff
@@ -0,0 +1,5 @@
+<div>
+  <div ng-if="$ctrl.footerIsDisplayed">
+    <oppia-footer></oppia-footer>
+  </div>
+</div>
```

Comment:
There should be an EOL here. (You might find it a good idea, after each PR, to check the "Files Changed" tab immediately once you submit to see what the reviewers see.)

Reply:
Sorry I missed it.  Done!